.concourse/Dockerfile

```diff
@@ -11,7 +11,7 @@ FROM    golang:1.11
 
 # Also installed docker, docker-compose, dockerize, jq
 
-ARG     K8S_VER=v1.13.4
+ARG     K8S_VER=v1.14.0
```

Comment:
FYI: on costinm/istio-installer I am trying to create a single "istio-builder" image set that can be used in all CI/CD systems, based on kindest/node ( i.e. the image used to run kind ). The image has built-in kubectl.

The goal would be to have  a set like istio-builder:k8s1.14, etc - so we can test multiple versions.